system_tests/workflow/test_end_to_end.py

```diff
@@ -154,6 +154,8 @@ def test_create(
         self._assert_job_exists(model_manager_client, job_id)
         # Get dataset ID used in this job
         job_resource = model_manager_client.read_job_by_id(job_id)
+        job_by_name = model_manager_client.read_job_by_model_name(model_name)
+        assert job_by_name is not None
```

Comment:
Not sure but maybe we can also test the `read_job_by_model_name` by replacing the lines 146-152 with a call to `read_job_by_model_name`?

Reply:
Yes, that would work! Updated.